store/src/main/java/org/apache/rocketmq/store/queue/BatchConsumeQueue.java

```diff
@@ -809,10 +818,11 @@ private MappedFile searchTimeFromFiles(long timestamp) {
     }
 
     /**
-     * Find the offset of which the value is equal or larger than the given targetValue.
-     * If there are many values equal to the target, then find the earliest one.
+     * Find the offset of which the value is equal or larger than the given targetValue. If there are many values equal
```

Comment:
As you are changing the doc, let's make it proper: 
If there are multiple values equal to the target, return the earliest one.